litmus-portal/backend/auth/pkg/jwt/jwtVerify.go

```diff
@@ -0,0 +1,39 @@
+package jwt
```

Comment:
What is the use of this function we already have a validate function https://github.com/litmuschaos/litmus/blob/76a4249159a3e409f0f66ae0a927b55b2c7595d9/litmus-portal/backend/auth/pkg/generates/jwt_access.go#L94

Reply:
Yes raise PR today, actually i developed that in earlier days for JWT Token coming from frontend as a pre check in every middleware.

Reply:
I believe @Sanjay1611 has already created a `validate` function in `auth` package, I don't think there any need to have another function do the same in the same package?

Reply:
@gdsoumya Yes talked with @Sanjay1611.